lms/static/styles/components/_Table.scss

```diff
@@ -15,6 +15,7 @@ $table-icon-size: 24px;
   border-collapse: collapse;
   width: 100%;
   font-size: $normal-font-size;
+  table-layout: fixed;
```

Comment:
This prevents the browser from trying to auto-size columns based on their contents. Instead the "Last modified" column specifies a fixed width and the name column gets the remainder. If the iframe happens to get larger or smaller in future, everything should still work.